.gitignore

```diff
@@ -49,6 +49,9 @@ bld/
 *.VisualState.xml
 TestResult.xml
 
+# Test project directory
+**/Projects/Temp/
+
```

Comment:
This will prevent test project files from flooding git tracking if unit testing is aborted for any reason.